client-js/app/scripts/app.ts

```diff
@@ -23,10 +25,11 @@ window.addEventListener('WebComponentsReady', function() {
     page(e.detail.path);
   });
 
-  var socket = new registry.JsonRpcSocket('ws://localhost:9000/api/socket2');
+  var socket = new clientRpc.JsonRpcSocket('ws://localhost:9000/api/socket2');
+  socket.prefix = "org.labrad."
   var reg = new registry.RegistryServiceJsonRpc(socket);
+  var echo = new echoServer.EchoServiceJsonRpc(socket);
```

Comment:
prefix is just for namespacing method names, so it would certainly be possible to serve requests with entirely different prefixes on the same server over the same websocket. It just seems like not a good idea to divide the task of constructing the full method name between this jsonrpc socket and the service implementations which use it.